server.js

```diff
@@ -7967,6 +7967,86 @@ cache(function (data, match, sendBadge, request) {
   });
 }));
 
+// Launchpad integration
+camp.route(/^\/launchpad\/ppa\/([^/]+)\/([^/]+)\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  // The type of badge, either version or downloads
+  var type = match[1];
+  // The owner of the ppa
+  var user = match[2];
+  // The name of the ppa
+  var ppa = match[3];
+  // The package in the PPA to use
+  var name = match[4];
+  var format = match[5];
+
+  var badgeData;
+
+  if (type == 'version') {
+    badgeData = getBadgeData('version', 'invalid');
+  }
+  else if (type == 'downloads') {
+    badgeData = getBadgeData('downloads 24h', 'invalid');
```

Comment:
Other downloads badges uses world `downloads` as a badge's left text. `24h` can be moved to badge's right text as `/day`.